people.go

```diff
@@ -389,3 +424,40 @@ func deletePerson(w http.ResponseWriter, r *http.Request) {
 	w.WriteHeader(http.StatusOK)
 	json.NewEncoder(w).Encode(true)
 }
+
+func getAssetByPubkey(pubkey string) ([]AssetBalanceData, error) {
+	client := &http.Client{}
+	testMode, err := strconv.ParseBool(os.Getenv("TEST_MODE"))
+	if err != nil {
+		testMode = false
+	}
+
+	url := os.Getenv(testModeUrl)
```

Comment:
we might want to rename this variable to be `LIQUID_URL` instead of `TEST_MODE` and then if a user wants to run their own liquid.sphinx.chat they can do so

Reply:
> we might want to rename this variable to be `LIQUID_URL` instead of `TEST_MODE` and then if a user wants to run their own liquid.sphinx.chat they can do so

@kevkevinpal the URL is meant for test mode alone; the prod URL is in the codebase already, and `liquidUr` l might not distinguish between test and prod. I guess `liquidTestModeUrl` might show the intention better. What do you think?

@Evanfeenstra will remove any unnecessary variables and re-push